src/pages/token-metadata/guides/javascript/create-an-nft.md

```diff
@@ -0,0 +1,449 @@
+---
+title: How to Create a NFT On Solana
+metaTitle: How to Create a NFT On Solana
+description: Learn how to create an NFT on the Solana blockchain with the Metaplex.
+# remember to update dates also in /components/guides/index.js
+created: '06-16-2024'
+updated: '06-18-2024'
+---
+
+This is an intial guide on how to create an NFT on the Solana blockchain with the Metaplex Token Metadata protocol.
+
+## Prerequisite
+
+- Code Editor of your choice (recomended Visual Studio Code)
+- Node 18.x.x or above.
+
+## Initial Setup
+
+This guide will run through create of an Nft Core Asset with Javascript based on a single file script. You may need to modify and move functions around to suit your needs.
```

Comment:
Based on offline discussion, we still need to change 'Nft' to 'NFT' throughout this PR.